LBHackney-IT/lbh-fss-portal-frontend
# add organisation status filter

**Description**

Add status filter dropdown to organisation listing page

**Test instructions**

- pull this branch 
- navigate to /organisations
- check status filter functionality works as expected, including combining the status filter with the search filter 

![image](https://user-images.githubusercontent.com/69238887/99105046-bf25f980-25d9-11eb-9fdb-8f593eb35ddf.png)


**Ticket**
https://app.clickup.com/t/a6ztz0
